packages/sdks/react-sdk/src/components/AuthProvider/AuthProvider.tsx

```diff
@@ -81,6 +93,20 @@ const AuthProvider: FC<IAuthProviderProps> = ({
   const isSessionFetched = useRef(false);
   const isUserFetched = useRef(false);
 
+  // if oidc config is enabled, and we have oidc params in the url
+  // we will finish the login (this should run only once)
+  useEffect(() => {
+    if (sdk && oidcConfig && hasOidcParamsInUrl() && !isOidcFinishedLogin.current) {
```

Comment:
Don't you think this logic should be part of the web-sdk?

Reply:
we can, but it implies that when the sdk is init, it will set the `isOidcLoading` state, which we don't want to do if its not required

Reply:
Not sure I understand, let's talk about it when you have 5 min